build/ps-entrypoint.sh

```diff
@@ -295,7 +295,7 @@ if [ "$1" = 'mysqld' -a -z "$wantHelp" ]; then
 			GRANT ALL ON *.* TO 'operator'@'${MYSQL_ROOT_HOST}' WITH GRANT OPTION ;
 
 			CREATE USER 'xtrabackup'@'localhost' IDENTIFIED BY '${XTRABACKUP_PASSWORD}';
-			GRANT SYSTEM_USER, BACKUP_ADMIN, PROCESS, RELOAD, LOCK TABLES, REPLICATION CLIENT ON *.* TO 'xtrabackup'@'localhost';
+			GRANT SYSTEM_USER, BACKUP_ADMIN, PROCESS, RELOAD, REPLICATION_SLAVE_ADMIN, LOCK TABLES, REPLICATION CLIENT ON *.* TO 'xtrabackup'@'localhost';
```

Comment:
Should we add `GROUP_REPLICATION_ADMIN` as well right now or not?
Docs say: `This privilege does not apply to Group Replication; use GROUP_REPLICATION_ADMIN for that.`
https://dev.mysql.com/doc/refman/8.0/en/privileges-provided.html#priv_replication-slave-admin